cypress/e2e/brewery/ScenarioViewDashboard.cy.js

```diff
@@ -93,4 +81,31 @@ describe('Scenario view PowerBI report', () => {
     ScenarioParameters.discard();
     Scenarios.checkIfReportIsUnsynced(true);
   });
+
+  it('If the back end returns a start time as null, out-of-synch is never displayed', () => {
+    const CURRENCY_VALUE_TO_UPDATE = 97779;
+    stub.setScenarioRunOptions({ startTime: null });
+
+    ScenarioParameters.expandParametersAccordion();
+    Scenarios.checkIfReportIsUnsynced(false);
+    Scenarios.getScenarioBackdrop(10).should('not.be.visible');
+    Scenarios.getDashboardAccordionLogsDownloadButton().should('not.exist');
+    Scenarios.checkIfReportIsUnsynced(false);
+
+    ScenarioParameters.launch({ scenarioId, runOptions, saveAndLaunch: true });
+    Scenarios.checkIfReportIsUnsynced(false);
+    Scenarios.getDashboardAccordionLogsDownloadButton().should('not.exist');
+    ScenarioParameters.waitForScenarioRunEnd();
+
+    Scenarios.checkIfReportIsUnsynced(false);
+    Scenarios.getDashboardAccordionLogsDownloadButton().should('be.visible');
+    Scenarios.checkIfReportIsUnsynced(false);
+    Scenarios.getDashboardAccordionLogsDownloadButton().should('be.visible');
+
+    Scenarios.checkIfReportIsUnsynced(false);
```

Comment:
The test is checking the same assertions several times, these lines can probably be removed :slightly_smiling_face: